src/Navbar/navbar.jsx

```diff
@@ -9,45 +9,44 @@ import MoTrPAClogo from '../assets/logo-motrpac.png';
  * Renders the global header nav bar.
  *
  * @param {Boolean}   isAuthenticated Redux state for user's authentication status.
- * @param {object}    profile         Redux state for authenticated user's info.
+ * @param {Object}    profile         Redux state for authenticated user's info.
  * @param {Function}  logout          Redux action for user logout.
  *
  * @returns {Object} JSX representation of the global header nav bar.
  */
 export function Navbar({
-  isAuthenticated = false,
+  isAuthenticated,
   profile,
   logout,
 }) {
-  const scrollFunction = () => {
-    if (document.body.scrollTop >= 30 || document.documentElement.scrollTop >= 30) {
-      document.querySelector('.navbar-brand').classList.add('resized');
-    } else {
-      document.querySelector('.navbar-brand').classList.remove('resized');
-    }
-  };
-
   const handleLogout = () => {
     logout();
     return <Redirect to="/" />;
   };
 
   if (isAuthenticated) {
-    window.removeEventListener('scroll', scrollFunction);
-  } else {
-    window.addEventListener('scroll', scrollFunction);
+    document.querySelector('body').classList.add('authenticated');
   }
 
   // Temp config to show/hide test interface visuals
   const urlParams = new URLSearchParams(window.location.search);
 
   // Function to render login button
   const LogoutButton = () => {
+    const userDisplayName = profile.user_metadata && profile.user_metadata.name
+      ? profile.user_metadata.name : profile.name;
+    const siteName = profile.user_metadata && profile.user_metadata.siteName
+      ? ', ' + profile.user_metadata.siteName : '';
```

Comment:
`eslint` is unhappy about this string concatenation

Reply:
Thank you for the great catch! I have addressed it in my latest commits.

Reply:
Thanks for converting it to a template.